src/app/plugins/leaderboard.plugin.ts

```diff
@@ -12,9 +12,7 @@ export class LeaderboardPlugin extends Plugin {
   public permission: ChannelType = ChannelType.Public;
   public pluginChannelName = Constants.Channels.Public.Games;
 
-  private readonly _MENTION_REGEX = /<@!?&?(\d+)>/;
-
-  public validate(message: IMessage, args: string[]) {
+  public validate(_message: IMessage, args: string[]) {
```

Comment:
why change `message` to `_message`

Reply:
In typescript you mark parameters you don't use by prefixing them with an underscore, this tells them to be skipped by the compiler.